components/oauth/org.wso2.carbon.identity.discovery/src/main/java/org/wso2/carbon/identity/discovery/DefaultOIDCProcessor.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (c) 2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.wso2.carbon.identity.discovery;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.wso2.carbon.base.ServerConfigurationException;
+import org.wso2.carbon.identity.discovery.builders.DefaultOIDProviderRequestBuilder;
+import org.wso2.carbon.identity.discovery.builders.OIDProviderRequestBuilder;
+import org.wso2.carbon.identity.discovery.builders.ProviderConfigBuilder;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+public class DefaultOIDCProcessor implements OIDCProcessor {
```

Comment:
Can we have class level javadoc on all the classes that are missing those? Please fix them throughout all your PRs.